src/entity/user/User.ts

```diff
@@ -88,6 +96,38 @@ export class User {
   @Index('users_github_unique', { unique: true })
   github?: string;
 
+  @Column({ length: 39, nullable: true })
```

Comment:
Are these set to 39 a max?

Reply:
yes, existing handles are the same so seems like reasonable limit